lucida/commandcenter/controllers/Config.py

```diff
@@ -12,29 +13,45 @@
 
 # Pre-configured services.
 # The ThriftClient assumes that the following services are running.
-# Host IP addresses are resolved dynamically: 
+# Host IP addresses are resolved dynamically:
 # either set by Kubernetes or localhost.
-SERVICES = { 
-	'IMM' : Service('IMM', 8082, 'image', 'image'), 
-	'QA' : Service('QA', 8083, 'text', 'text'),
-	'CA' : Service('CA', 8084, 'text', None),
-	'IMC' : Service('IMC', 8085, 'image', None),
-	'FACE' : Service('FACE', 8086, 'image', None),
-	'DIG' : Service('DIG', 8087, 'image', None),
-	'ENSEMBLE' : Service('ENSEMBLE', 9090, 'text', None) 
-	}
+SERVICES = {
+    'IMM' : Service('IMM', 8082, 'image', 'image'),
```

Comment:
I'm thinking we could using something like this https://github.com/Netflix/eureka/wiki/Eureka-at-a-glance where there is a server (contains the registry of services) and a client (within the cmd center/microservices). All clients register its network location using a POST request and retrieve the registered services through an HTTP GET request. It would be best if I opened an issue to contain this feature instead of including it in this PR. What do you think?